### PR TITLE
edit Week1/homework/app.js and edit Week1/homework/index.html and add…

### PR DESCRIPTION
**Pull request by abdulsalamalmahdi**
… js2_week1_class exercise/index.html and add js2_week1_class exercise/index.js

### CODE REVIEW

**Comment by abdulsalamalmahdi** [comment]
could not figure out how to insert the object into list items

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
+    'a_tale_of_two_cities',
+    'the_lord_of_the_rings',
+    'harry_potter_and_the_secret_stone',
+    'and_then_there_were_none',
+    'the_hobbit'
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  function idToListF (array){
```
Comment (bitsagarob) [inline Week1/homework/app.js]: try to come up with a better name, make clear what it does

File: js2_week1_class exercise/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const addToDoBtn = document.getElementById('btn-addTodo');
+
+const changeAddToDoF = () => {
+   
+   var todoInput = document.getElementById('todoInput').value;
+    listNode = document.getElementById('todoList');
+    liNode = document.createElement('LI');
+    textNode = document.createTextNode(todoInput);
+    const todoList = listNode.removeChild(listNode.getElementsByTagName('li')[0]);
```
Comment (bitsagarob) [inline js2_week1_class exercise/index.js:22]: try to get the list by ID

File: js2_week1_class exercise/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    liNode.appendChild(textNode);
+    listNode.appendChild(liNode);
+    
+  }
+  addToDoBtn.addEventListener("click", () => changeAddToDoF());
+
```
Comment (bitsagarob) [inline js2_week1_class exercise/index.js:28]: is there a missing } here?

**Review by bitsagarob — CHANGES_REQUESTED** [review]
Nice try, good job! Is this code working? there seem to be syntax error. Try to make the books exercise also

**Pull request closed without merging**